storage/txcache/sweeping_test.go

```diff
@@ -23,29 +23,30 @@ func TestSweeping_CollectSweepable(t *testing.T) {
 	cache.NotifyAccountNonce([]byte("bob"), 20)
 
 	// 1st fail
-	selection = cache.SelectTransactions(1000, 1000)
+	selection = cache.doSelectTransactions(1000, 1000)
 	require.Equal(t, 1, len(selection))
 	require.Equal(t, 0, len(cache.sweepingListOfSenders))
 	require.Equal(t, 1, cache.getNumFailedSelectionsOfSender("alice"))
 	require.Equal(t, 1, cache.getNumFailedSelectionsOfSender("bob"))
 	require.Equal(t, 0, cache.getNumFailedSelectionsOfSender("carol"))
 
 	// 2nd fail, grace period, one grace transaction for Alice and Bob
-	selection = cache.SelectTransactions(1000, 1000)
+	selection = cache.doSelectTransactions(1000, 1000)
 	require.Equal(t, 3, len(selection))
 	require.Equal(t, 0, len(cache.sweepingListOfSenders))
 	require.Equal(t, 2, cache.getNumFailedSelectionsOfSender("alice"))
 	require.Equal(t, 2, cache.getNumFailedSelectionsOfSender("bob"))
 	require.Equal(t, 0, cache.getNumFailedSelectionsOfSender("carol"))
 
 	// 3nd fail, collect Alice and Bob as sweepables
-	selection = cache.SelectTransactions(1000, 1000)
+	selection = cache.doSelectTransactions(1000, 1000)
+	// o si rulat gorutina si a sters offfffff. cum pot oare?
```

Comment:
is the comment still valid?
check below as well

Reply:
Thanks! Fixed / removed.